mtp_cashbook/assets-src/stylesheets/views/_batch.scss

```diff
@@ -3,12 +3,17 @@
   margin-bottom: 1em;
 
   p {
-    display: inline;
     font-size: 24px;
+    margin-top: 0;
+
+    @include breakpoint(769px) {
```

Comment:
Why `769px`? Is there some way we can make it clearer what this breakpoint is for?